src/animations/loading-spinner/loading-spinner.tsx

```diff
@@ -17,5 +17,5 @@ export const LoadingSpinner = (props: BaseAnimationProps) => {
 
 const Container = styled.div`
     margin: 0 auto;
-    padding: 1rem 2rem 4rem;
+    padding: 3rem 2rem 2rem;
```

Comment:
the original padding was meant to visually center the spinner vertically. I think this can be reduced with

`padding: 0 1rem 0;`